python/lsst/pipe/base/pipelineTask.py

```diff
@@ -59,9 +59,12 @@ class DatasetTypeDescriptor:
     datasetType : `DatasetType`
     scalar : `bool`
         `True` if this is a scalar dataset.
+    manualLoad : `bool`
+        `True` if this dataset will be manually loaded by a concrete
+        `PipelineTask` instead of loaded automatically by the base class.
     """
 
-    def __init__(self, datasetType, scalar, manualLoad):
+    def __init__(self, datasetType, scalar, manualLoad, expected=False):
```

Comment:
Accidental relic of an earlier draft, when "expected" was the term I was using for what is now "prerequisite", and the label was part of `DatasetTypeDescriptor` instead of a new `PipelineTask` `classmethod`.